v3/src/components/graph/models/graph-controller.ts

```diff
@@ -62,22 +62,25 @@ export class GraphController {
   initializeGraph() {
     const {graphModel, layout, dotsRef, enableAnimation, instanceId, v2Document} = this,
       dataConfig = graphModel.config
+
     if (v2Document) {
       this.processV2Document()
     } else {
-      graphModel.setGraphProperties({
-        axes: {bottom: EmptyAxisModel.create({place: 'bottom'}),
+      if (!dotsRef.current){
```

Comment:
combine this with line above:
```suggestion
      } else if (!dotsRef.current) {
```

Reply:
If I do this TypeScript complains `Property 'processV2Document' does not exist on type 'GraphController'.ts(2339)`

Reply:
When looking at the GitHub differences I thought my suggestion was equivalent, but now I see that it's not, so forget what I said. 😉